helpers/tests/s3.rs

```diff
@@ -0,0 +1,96 @@
+#![cfg(feature = "s3")]
```

Comment:
I need to yank this whole file out.  I moved the S3Helper when I started the rewrite, but forgot it was there.  This almost certainly doesn't compile or work.